CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 ## Release History
+### 1.15.0 (2024-04-17)
```

Comment:
if we are going to release today, we should update the date for today.

Reply:
changed